app/controllers/omniauth_callbacks_controller.rb

```diff
@@ -14,15 +14,16 @@ def github
 
   def failure
     error = request.env["omniauth.error"]
+    class_name = error.present? ? error.class.name : ""
```

Comment:
Do you have an example of what happens when this fails but there is no error? Just curious

Reply:
Nope, just doing some defensive programming, which can be removed. I don't think it can happens, except in tests when the mocking is not done (that's how i triggered that behavior, at the start of the PR)

Reply:
What you saw in the test probably happened because of the reliance on implicit `to_s` we talked about before. But I guess "" is still better than "NilClass".  :smiley: